src/main/java/org/sagebionetworks/bridge/reporter/worker/RetentionReportGenerator.java

```diff
@@ -65,20 +74,27 @@ public Report generate(BridgeReporterRequest request, String studyId) throws IOE
                         break;
                     }
                 }
+                
                 if (studyStartDate == null) {
                     LOG.error("No study_state_date event for id=" + accountSummary.getId());
                     continue;
                 }
-                RequestInfo requestInfo = bridgeHelper.getRequestInfoForParticipant(studyId, accountSummary.getId());
+                
+                RequestInfo requestInfo = bridgeHelper.getRequestInfoForParticipant(
+                        studyId, accountSummary.getId());
                 if (requestInfo.getSignedInOn() != null) {
-                    int sign_in_days = Days.daysBetween(studyStartDate.toLocalDate(), requestInfo.getSignedInOn().toLocalDate()).getDays();
+                    int sign_in_days = Days.daysBetween(studyStartDate, 
+                            requestInfo.getSignedInOn().withZone(DateTimeZone.UTC)).getDays();
```

Comment:
Even though studyStartDate is always in UTC, we should explicitly set the time zone here, just to be safe. (Also when we calculate upload_on_days.)